src/Nix/Diff/Types.hs

```diff
@@ -77,6 +76,9 @@ type Builder = Text
 
 type Argument = Text
 
+-- | A set of Nix derivation output names.
+type Outputs = Set Text
```

Comment:
Usually I try to avoid type synonyms, preferring either no synonym or a newtype.  The reason why is because they will not reliably show up in inferred types or type errors.  Also, `Set Text` is not that much longer than `Outputs` (even with parentheses)

Alternatively, I would accept a change to make this a `newtype`, although that would be a breaking change

Reply:
Done!